### ISSUE TITLE
Bug: Right click broken in 6.7.9 text area

### ISSUE THREAD

**Issue by Tarun80** [notepad-plus-plus/notepad-plus-plus#217]
If you try to right click anywhere in the text editor area the context menu does not appear. Right click functionality works in the search results.

**Comment by Tarun80** [comment]
After restarting my computer, the right click context menu started to work again. The only change I made was close 6.7.8.2 to update to 6.7.9. Right click did not work after that until the restart, strangely enough.

**Comment by Srekel** [comment]
This happens to me and I'm on 6.7.5. Restarting notepad is the only way I've found that fixes the problem.

I did have a pretty solid repro case for it: if I did a find in files and I had a huge file in my project (50mb lua file) it'd stop working after that. I've since removed my file but it still happens occasionally, and I'm not sure if it's connected to find in files or not. I do it very frequently and the bug seems to appear randomly.

It's super annoying so a fix would be greatly appreciated :)

**Comment by Srekel** [comment]
Ok, I've now upgraded to 6.7.9.2. According to the release notes for 6.7.9 there was a fix for the context menu after using Find in files, but I now have a solid repro case where I can start up the program, do a find in files search, and after it's done the context menu doesn't work. 

Also note that it's not always like this, occasionally it seems like I can work for a while and the context menu will work after a find in files, so maybe it depends on something else I've done previously.

Note that I have a pretty big project (probably 10000 files, about 1000 of which are Lua files which is what my search filter is set to). The first time I noticed this happening was, as I mentioned, after I added a really big Lua file to the project folder.
